config/database.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+$heroku_db_url = parse_url(env('DATABASE_URL', "postgres://uynuxroikxovrl:c6b3d9a507870994d5ef8de5c76d56498d2e06b0da4876d85db77a418a6b5b16@ec2-54-247-81-88.eu-west-1.compute.amazonaws.com:5432/dd931vrcm2t44j"));
```

Comment:
This should be put as an environment variable. For security reasons, it is not advisable you hardcode sensitive information in your application and push to a version control.

By default, Lumen uses dotenv in development. you should take advantage of that